src/pages/AddMembersPage/AddMembersPage.tsx

```diff
@@ -157,13 +155,39 @@ const AddMembersContents = ({
           </Box>
         )}
         {currentTab === Tab.LINK && (
-          <>
-            <div>
-              MagicLink
+          <Box>
+            <Text css={{ pb: '$lg', pt: '$sm' }} size={'large'}>
```

Comment:
My IDE auto does this, I just found the setting to prevent it , but its not something i can just fix easily without a fine tooth comb... guess i'll go that :|

Reply:
fixed all this i think